### PR TITLE
Selenium 3 + chrome 79 (december 2019)

### PR DESCRIPTION
**Pull request by ahocquard**
**Description (for Contributor and Core Developer)**

This PR uses Selenium 3 and a recent Chrome. The previous Firefox version is outdated (2016...) and only represent 0.2% of people.

I waste A LOT of timeon stabilization. Maybe there will still be some unstables left, but I definitively can't fix it alone. I let the team to decide if future unstable tests should be deleted.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by ahocquard** [comment]
@juliensnz I agree. I will let you define that with the blackhawks (it's my last day in this team, RIP :smile: ). For now, I think migration to chrome + selenium 3 is a good step. Fixing the version number of chrome to a lower version is a second step.
I think it's not a big deal.

File: docker-compose.override.yml.backup [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: '3.4'
```
Comment (nmarniesse) [inline docker-compose.override.yml.backup]: Is this file committed on purpose?

File: behat.yml [hunk 2]
```diff
@@ -156,6 +157,7 @@ legacy:
                 selenium2:
                     selenium2:
                         wd_host: 'http://selenium:4444/wd/hub'
+                        capabilities: { "browserName": "chrome", "browser": "chrome", 'chrome': {'switches':['--no-sandbox']}}
```
Comment (nmarniesse) [inline behat.yml]:
```suggestion
                        capabilities: { "browserName": "chrome", "browser": "chrome", "chrome": {"switches":["--no-sandbox"]}}
```

JSON-like values with single quotes annoys me :smile:

File: tests/legacy/features/Context/DataGridContext.php [hunk 3]
```diff
@@ -451,6 +451,7 @@ public function iDisplayTheColumns($gridLabel, $columns)
      */
     public function iShouldSeeTheColumns($columns)
     {
+        sleep(30);
```
Comment (nmarniesse) [inline tests/legacy/features/Context/DataGridContext.php]: 30 seconds is very long.. Are you sure it's necessary, is it too unstable?

Comment (ahocquard) [reply tests/legacy/features/Context/DataGridContext.php]: debug, nice catch

**Pull request merged**